apps/jobs/urls.py

```diff
@@ -85,4 +85,14 @@
         views.get_github_badge_data,
         name="get_github_badge_data",
     ),
+    url(
+        r"^challenge/(?P<challenge_pk>[0-9]+)/challenge_phase/(?P<challenge_phase_pk>[0-9]+)/get_presigned_url_for_submission/$",
+        views.get_presigned_url_for_submission,
+        name="get_presigned_url_for_submission",
+    ),
+    url(
+        r"^publish_submission_message/$",
```

Comment:
@RishabhJain2018  can you confirm if this API url is okay?

Reply:
I don't think it is as it needs to have all the checks and should accept phase_pk parameter.
cc: @Ram81